tests/models/wav2vec2/test_modeling_flax_wav2vec2.py

```diff
@@ -302,6 +304,11 @@ def test_model_from_pretrained(self):
             outputs = model(np.ones((1, 1024), dtype="f4"))
             self.assertIsNotNone(outputs)
 
+    @is_pt_flax_cross_test
+    @is_flaky
```

Comment:
This should be `@is_flaky()`, otherwise the actual test won't be run and it always passes.
(hopefully I am doing right in my experiments)